ItalianTranslation_PlanetCoaster.xml

```diff
@@ -13926,25 +13926,25 @@
 		<entry key="OptionsMenu_Controls_Path_Edit_Command_01" translation="Posiziona Sentiero"/>
 		<entry key="OptionsMenu_Controls_Path_Edit_Command_02" translation="Elimina Sentiero"/>
 		<entry key="OptionsMenu_Controls_Path_Edit_Command_03" translation="Alza/Abbassa Sentiero"/>
-		<entry key="OptionsMenu_Controls_Path_Edit_Command_04" translation="Alza/Abbassa Sentiero (Senza Posizionarlo)"/>
```

Comment:
Sei sicuro che lo alzi soltanto? Non fai su/giù? (Ora non ricordo)

Reply:
Ho verificato la traduzione in inglese 1.39 e hanno modificato le voci (cambiato l'ordine e aggiunto voci diverse), quindi mi sono basato su questa.

Reply:
D'accordo